vocabulary/wd/index.html

```diff
@@ -46,17 +52,17 @@ <h1 property="dc:title schema:name">W3C Web Annotation vocabulary</h1>
 <dl><dt>Imported Ontologies:</dt>
 <dd><a property="owl:imports schema:mentions" resource="http://www.w3.org/ns/prov#" href="http://www.w3.org/ns/prov#">PROV-O</a></dd><dd><a property="owl:imports schema:mentions" resource=" http://www.w3.org/2004/02/skos/core" href=" http://www.w3.org/2004/02/skos/core"> SKOS</a></dd></dl>
 
-<dl>This work is licensed under a <a rel="license" href="http://www.w3.org/Consortium/Legal/copyright-documents">Copyright © 2014 W3C® (MIT, ERCIM, Keio, Beihang), All Rights Reserved. W3C liability, trademark and document use rules apply.</a>.</dl>
-<span property="dc:license" resource="http://www.w3.org/Consortium/Legal/copyright-documents"></span>
-<hr/>
+<dl>This work is licensed under a license: <a rel="license" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document">Copyright © 2016 W3C ® (MIT, ERCIM, Keio, Beihang). W3C liability, trademark use rules apply. This Document is licensed under the W3C Software and Document license</a>.</dl>
```

Comment:
This is incorrect, please re-establish what was there before. The recommendation's license is what it used to be (except of course for the date), and there is no place for the software license at this point